broker/src/test/resources/system/feature-flags-cfg.yaml

```diff
@@ -6,3 +6,4 @@ zeebe:
         enableActorMetrics: true
         enableMessageTTLCheckerAsync: true
         enableTimerDueDateCheckerAsync: true
+        enableStraightThroughProcessingLoopDetector: false
```

Comment:
:wrench: I was wondering before that this was set to false, which is also why I marked the other cases of wrong naming. I guess we want to have true here then?